lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetCompressor.java

```diff
@@ -43,13 +45,14 @@ static ParquetCompressor getCompressor(CompressionCodec codec)
                 return new AirLiftCompressor(new ZstdCompressor());
             case UNCOMPRESSED:
                 return null;
-            case LZO:
             case LZ4:
+                // Note: LZ4 compression scheme has been deprecated by parquet-format in favor of LZ4_RAW.
+                // However, LZ4 is the more widely supported encoding in other engines.
+                return new Lz4FramedCompressor();
+            // TODO Support LZO and LZ4_RAW compression
```

Comment:
Let's remove the TODO. It doesn't seem like this is something we want to implement now, and there's no straightforward path to do so (since it doesn't map 1:1 with the Hive compression enum)

Reply:
removed the TODO